triggers/daml/Daml/Trigger.daml

```diff
@@ -130,7 +144,7 @@ dedupExerciseByKey k c = do
   -- map to make these lookups cheaper.
   let cmds = concat $ map snd (Map.toList aState.commandsInFlight) <> map commands aState.emittedCommands
   unless (any ((Some (k, c) ==) . fromExerciseByKey @t) cmds) $
-    void $ emitCommands [exerciseByKeyCmd @t k c]
+    void $ emitCommands [exerciseByKeyCmd @t k c] []
```

Comment:
Why does `dedupExerciseByKey` behave differently from `dedupExercise` w.r.t. pening-set?

Reply:
See comment below, they now both behave the same.